website/docs/docs/dbt-versions/release-notes/04-Sept-2023/removing-prerelease-versions.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Removing old (prerelease) versions of dbt from dbt Cloud when (latest) is available"
+description: "Improving the version selection options by removing prerelease versions whenever the latest version is available."
+sidebar_label: "Removing old prerelease versions from dbt Cloud"
+tags: [Sept-2023, Versions]
+date: 2023-09-26
+sidebar_position: 07
+---
+
+Previously, when dbt released a new [version](/docs/dbt-versions/core#how-dbt-core-uses-semantic-versioning) in dbt Cloud, the older patch _prerelease_ version, in addition to the _latest_, remained as an option in the dropdown menu available in the **Environment settings**. Now, when the _latest_ version is released, the _prerelease_ version will be removed, and all customers remaining on it will be migrated seamlessly. There will be no interruptions to service when this migration occurs. 
+
+To see which version you are currently using, and to upgrade, navigate to the **Deploy** option in the menu bar and click **Environments**. Click into the preferred environment and click **Settings**. Click **Edit** to make a change to the current dbt version.  dbt Labs recommends always using the latest version whenever possible to take advantage of new features and functionality. 
```

Comment:
per style guide, we use "click" for buttons and "select/choose" for menus, etc.

```suggestion
To see which version you are currently using and to upgrade, select **Deploy** in the top navigation bar and select **Environments**. Choose the preferred environment and click **Settings**. Click **Edit** to make a change to the current dbt version. dbt Labs recommends always using the latest version whenever possible to take advantage of new features and functionality. 
```

Reply:
Do we call it a "navigation bar"? is that the official term? I've seen (and been using) menu bar. Do you know if there is official guidance on this?

Reply:
not sure tbh. i've been using top nav bar 🤷‍♀️